src/menus.tsx

```diff
@@ -104,6 +108,102 @@ export const getEditorMenus = (
     label: 'Insert',
     isEnabled: true,
     submenu: [
+      {
+        id: 'front-matter',
+        label: 'Article Metadata',
+        isEnabled: true,
+        submenu: [
+          {
+            id: 'insert-abstract',
+            label: 'Abstract',
+            isEnabled: isCommandValid(insertAbstract),
+            run: doCommand(insertAbstract),
+          },
+          {
+            id: 'insert-graphical-abstract',
+            label: 'Graphical Abstract',
+            isEnabled: isCommandValid(insertGraphicalAbstract),
+            run: doCommand(insertGraphicalAbstract),
+          },
+          {
+            id: 'insert-contributors',
+            label: 'Authors & Affiliations',
+            isEnabled: isCommandValid(insertContributors),
+            run: doCommand(insertContributors),
+          },
+          {
+            id: 'insert-keywords',
+            label: 'Keywords',
+            isEnabled: isCommandValid(insertKeywords),
+            run: doCommand(insertKeywords),
+          },
+        ],
+      },
+      {
+        id: 'back-matter',
+        label: 'Author Notes',
```

Comment:
its a new changes requested by Panos, but I have kept the id as back-matter since its more clear from code perspective. do you want to change it also?

Reply:
Understood. No need to change it.